test/4_fund_actions.js

```diff
@@ -127,37 +127,38 @@ contract('Fund Actions', (accounts) => {
           .then(success => assert.isTrue(success))
           .then(() => fund.modifyAllocation(investor, amt, { from: MANAGER }))
           .then(() => fund.getAvailableAllocation(investor))
-          .then((_allocation) => fund.getInvestor(investor))
+          .then((_allocation) => {
```

Comment:
This is an example of a case where since you are performing other actions before returning the promise, you need the curly brackets.  Since there is an operation `allocation = _allocation` before you move on to return the next promise `return getInvestor()`